presto-ui/src/components/StageDetail.jsx

```diff
@@ -432,6 +430,10 @@ const StageOperatorGraph = ({ stage }) => {
         }
     }, [stage, computeOperatorGraphs, computeD3StageOperatorGraph, handleOperatorClick]);
 
+    useEffect(() => {
```

Comment:
This effect was called conditionally, moved it up in the function so its always called. Worth a manual test to make sure the stage graph does not error bc of it